docs/RELEASE.md

```diff
@@ -0,0 +1,54 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Prepare release
+1. [ ]  User testing on mockups (design)
+1. [ ]  go/nogo on individual features (engineering)
+1. [ ]  create release tag for the selected features
+1. [ ]  all automated tests pass
+1. [ ]  all internationalization tests pass
+1. [ ]  generate release notes
+
+## Security checks
+TBD: Andrew will send a list
```

Comment:
File an issue on him, link the issue with a TODO.

Reply:
Will do.